.github/workflows/CI.yml

```diff
@@ -40,29 +40,14 @@ jobs:
         env:
           CI: true
 
-      # Deploy preview environment for non-forks
-      - uses: Azure/static-web-apps-deploy@v1
-        if: github.event.pull_request.base.repo.id == github.event.pull_request.head.repo.id
-        with:
-          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_PROD }}
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          action: upload
-          app_location: site
-          skip_app_build: true
-
-      # Upload site artifact for forks so it can be deployed by a maintainer on-demand
-      - uses: actions/upload-artifact@v3
-        if: github.event.pull_request.base.repo.id != github.event.pull_request.head.repo.id
```

Comment:
Er, is this not the code which powers the `deploy-preview` label? https://github.com/microsoft/TypeScript-Website/blob/b03935a55750acfadc496e5fdf4258730c96cb5f/.github/workflows/deploy-preview.yml

Probably, just the deploy is what needs to be deleted?

Reply:
Oh yeah, I thought it was doing a more native version of it, will drop next time I get back to the computer